src/app/student-dash/student-dash.component.html

```diff
@@ -69,13 +115,8 @@
   </div>
 
   <br>
-  <a
-    style="padding: 4px; float: right; text-align: right;"
-    (mouseenter)="toggleOK()"
-    (mouseleave)="toggleOK()"
-    [@testAnim]="ok"
-    href="https://github.com/Tattvum/edutvum-exam"
-    target="_blank"
-    >https://github.com/Tattvum/edutvum-exam</a>
+  <a style="padding: 4px; float: right; text-align: right;" (mouseenter)="toggleOK()" (mouseleave)="toggleOK()"
```

Comment:
Please avoid dispensable formatting. This is to aid clarity of changes required for the feature.